form.tex

```diff
@@ -0,0 +1,92 @@
+\documentclass[a4paper]{article}
+
+\usepackage[bmargin=3cm, tmargin=2cm]{geometry}
+\usepackage{graphicx}
+\usepackage{xspace}
+\usepackage[latin1]{inputenc}
+\usepackage[pdftex]{hyperref}
+\setlength\parindent{0pt}
+
+\usepackage{helvet}
+\renewcommand*\familydefault{\sfdefault} %% Only if the base font of the document is to be sans serif
+\usepackage[T1]{fontenc}
+
+\pagenumbering{gobble}
+
+%%%% Defines for form filling
+\newcommand{\signature}{%
+    Signature:\\
+    \vspace{1cm}\\
+    \rule{7cm}{1pt}\\
+    Date: \vbox{\hbox{\TextField[width=6cm]{\null}}\kern2pt\hrule}
+}
+
+\begin{document}
+
+\newcommand{\compyear}{2018\xspace}
+\newcommand{\competition}{SourceBots \compyear}
+
+\begin{minipage}[c]{0.30\textwidth}
+\includegraphics[height=2cm]{fig-SourceBots}
+\end{minipage}
+\hspace{0.03\textwidth}
+\begin{minipage}[c]{0.67\textwidth}
+\begin{center}
+{\large \textbf{\competition Media Consent Release Form} \par}
+\end{center}
+\end{minipage}
+\vspace{1cm}
+
+\textbf{This is not a ticket.}\\\\ However, this form is required for entry into the \competition competition.\\
+\vspace{0.3cm}
+
+\textbf{\large Media Consent}\\
```

Comment:
We should clarify that this consent applies only to recordings at the current event. This ends up being implied by the privacy policy at the moment, but I think we should be precise here and then be less specific in the privacy policy.

Reply:
fixed in 6ff922efd07757169a82cb4d3bc272783c720d49